pycaret/tests/test_time_series.py

```diff
@@ -145,3 +145,26 @@ def test_tune_model(model, load_data):
 
     expected_period_index = load_data.iloc[-fh:].index
     assert np.all(y_pred.index == expected_period_index)
+
+
+@pytest.mark.parametrize("model", _all_models)
+def test_tune_model_random(model, load_data):
+
+    from pycaret.internal.PycaretExperiment import TimeSeriesExperiment
+    exp = TimeSeriesExperiment()
+    fh = 12
+
+    exp.setup(
+        data=load_data,
+        fold=3,
+        fh=fh,
+        fold_strategy="expandingwindow"
+    )
+
+    model_obj = exp.create_model(model)
+    tuned_model_obj = exp.tune_model(model_obj, search_algorithm="random")
+    y_pred = tuned_model_obj.predict()
+    assert isinstance(y_pred, pd.Series)
+
+    expected_period_index = load_data.iloc[-fh:].index
```

Comment:
perhaps we can also test some behavior here, that the predictions from tune model have lower metric value than the predictions from the original model.

Reply:
Yes, I will add that when the flow is working from end to end and hyperparameter grid and choice has been tuned. Just getting the skeleton setup for now so this is a very basic unit test to make sure it runs.

Reply:
awesome